src/schema/v2/user.ts

```diff
@@ -534,6 +534,12 @@ export const UserType = new GraphQLObjectType<any, ResolverContext>({
         resolve: ({ receive_partner_show_notification }) =>
           receive_partner_show_notification,
       },
+      receivePartnerOfferNotification: {
```

Comment:
This field on the user type might not be needed technically for this ticket (UI toggle in Eigen), but I assume it's needed later in Pulse to determine if sending a push to a user or not.